src/parser.c

```diff
@@ -414,8 +408,12 @@ static void parseHeaderLine(flightLog_t *log, mmapStream_t *stream)
         log->sysConfig.currentMeterOffset = currentMeterParams[0];
         log->sysConfig.currentMeterScale = currentMeterParams[1];
     } else if (strcmp(fieldName, "gyro.scale") == 0 || strcmp(fieldName, "gyro_scale") == 0) {
-        floatConvert.u = strtoul(fieldValue, 0, 16);
+        union {
+        float f;
+        uint32_t u;
```

Comment:
Indentation. Also, having a type definition in the middle of code is unusual, I think this should go at the beginning of the file.